core/src/style/pointStyleBuilder.cpp

```diff
@@ -105,13 +105,6 @@ void PointStyleBuilder::setup(const Marker& _marker, int zoom) {
 }
 
 bool PointStyleBuilder::checkRule(const DrawRule& _rule) const {
```

Comment:
What is the reason of always making point style pass irrespective of any color or texture parameter set?

Reply:
Updated checkRule to also check dynamic textures. Thanks!